docs/2-deployments/7-observability.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Observability
+---
+# Observability
+Up until now, you have used Architect to develop and deploy your application. However, Architect's role in your development process does not end there. You still need to monitor the system and be able to dive in when a problem occurs. To help facilitate this, Architect provides some helpful commands that will allow you better insights into how your system is doing.
+
+Whether it be a remote deployment using [`architect deploy`](/reference/cli/#architect-deploy-configs_or_components) or a local one using [`architect dev`](http://localhost:8000/reference/cli/#architect-dev-configs_or_components) these commands will give you a deeper insight into your product.
+
+We will assume that you have deployed the example react app to a remote environment for the following examples.
+```sh
+$ architect register ./examples/react-app/
+$ Architect deploy react-app
+```
+
+## Logging
+
+As your application runs, it will generally print important information to stdout and stderr as it runs. The log command will allow you to stream those logs from a remote service to your console. For instance, let's say you wanted to see the latest logs for the API of the example react app. You would do the following.
+```sh
+$ architect logs -a my_account -e my_environment react-app.services.api
```

Comment:
Should we also showcase how to use this commands for local dev?

Reply:
Added in a new section that goes over the difference between Remote and Local. Also moved the guidance around what to do if you have no idea about what accounts or envs you have there.